### PR TITLE
Pre test balancing

### PR DESCRIPTION
**Pull request by LiamMillerSE**

### CODE REVIEW

File: Assets/Scripts/Audio/Jukebox.cs [hunk 1]
```diff
@@ -16,9 +16,12 @@ public class Jukebox : MonoBehaviour, IResettable
     public WaveSequence sequence;
     // An audiosource array with 2 members to switch between with "toggle"
     public AudioSource[] audioSourceArray;
+    // TODO: Do not make dependant on changeScene script
```
Comment (toryjleo) [inline Assets/Scripts/Audio/Jukebox.cs:19]: Address TODO

File: Assets/Scripts/Audio/Jukebox.cs [hunk 2]
```diff
@@ -82,8 +95,38 @@ public void ResetGameObject()
...
         {
             audioSourceArray[i].Stop();
+            audioSourceArray[i].clip = null;
         }
         sequence = soundTracks[Random.Range(0, soundTracks.Count)];
         Start();
     }
+
+    // TODO: try to move this logic out
+    private void LoadMainMenu()
```
Comment (toryjleo) [inline Assets/Scripts/Audio/Jukebox.cs:105]: Remove this (we will have a cinematic wave sequence)

File: Assets/Scripts/Enemies/Ai.cs [hunk 3]
```diff
@@ -113,7 +113,7 @@ public virtual void Die()
                 desiredVec *= dMag;
 
             //Attack();
```
Comment (toryjleo) [inline Assets/Scripts/Enemies/Ai.cs:115]: Remove commented out method

File: Assets/Scripts/Enemies/AiClasses/RangerAI.cs [hunk 4]
```diff
@@ -9,13 +9,13 @@ public class RangerAI : InfantryAI
     public override void Init()
     {
         alive = true;
-        StartingHP = 20;
+        /*StartingHP = 20;
```
Comment (toryjleo) [inline Assets/Scripts/Enemies/AiClasses/RangerAI.cs:12]: Delete commented out var setting

File: Assets/Scripts/Enemies/AiClasses/RiflemanAi.cs [hunk 5]
```diff
@@ -10,13 +10,13 @@ public class RiflemanAi : InfantryAI
...
     {
         alive = true;
-        StartingHP = 20;
-        score = 100;
-        dlScore = 5;
-        maxSpeed = 80;
-        attackRange = 60;
-        minimumRange = 0;
-        speedBoost = 20;
+        //StartingHP = 20;
```
Comment (toryjleo) [inline Assets/Scripts/Enemies/AiClasses/RiflemanAi.cs:13]: ditto

File: Assets/Scripts/Enemies/AiClasses/Shotgunner.cs [hunk 6]
```diff
@@ -2,20 +2,20 @@
...
-public class GruntAI : InfantryAI
+public class Shotgunner : InfantryAI
 {
     public GameObject muzzleLocation; // Empty GameObject set to the location of the barrel
 
     public override void Init()
     {
         alive = true;
-        StartingHP = 40;
+        /*StartingHP = 40;
```
Comment (toryjleo) [inline Assets/Scripts/Enemies/AiClasses/Shotgunner.cs:12]: ditto to above

File: Assets/Scripts/Enemies/VehicleAI.cs [hunk 7]
```diff
@@ -112,7 +112,7 @@ void OnCollisionEnter(Collision collision)
             Vector3 bumpForce = Vector3.ClampMagnitude((vehicleController.carVelocity - 
                 bike.movementComponent.appliedForce) * RamModifier, MAX_RAM_MAGNITUDE);
 
-            bikeRB.AddForce(bumpForce, ForceMode.Impulse);
+            //bikeRB.AddForce(bumpForce, ForceMode.Impulse);
```
Comment (toryjleo) [inline Assets/Scripts/Enemies/VehicleAI.cs:115]: remove commented out logic

File: Assets/Scripts/Player/BikeMovementComponent.cs [hunk 8]
```diff
@@ -128,14 +138,14 @@ public void ApplyForces()
 
         
         //Latteral movement inputs, q and e add side force to the bike
```
Comment (toryjleo) [inline Assets/Scripts/Player/BikeMovementComponent.cs:140]: remove unused lines (can always add later)

File: Assets/Scripts/Player/BikeScript.cs [hunk 9]
```diff
@@ -200,7 +200,24 @@ public void ResetGameObject()
     private void Dead() 
     {
         GameStateController.WorldState = GameState.Spawning;
+        //will make bike "ragdoll" but the camera has an anneurism so we'll leave it out for now
```
Comment (toryjleo) [inline Assets/Scripts/Player/BikeScript.cs:203]: If you want to keep this, move to method

File: Assets/Scripts/SceneManageMent/Waves/WaveSequence.cs [hunk 10]
```diff
@@ -10,33 +11,54 @@
...
+    }
+
+    internal void Init(SquadSpawner squadSpawner)
+    {
+        spawner = squadSpawner;
+        currentWave = 0;
+        previousWave = -1;
+    }
+
+    internal double GetNextWaveTime(double currentTime)
```
Comment (toryjleo) [inline Assets/Scripts/SceneManageMent/Waves/WaveSequence.cs:58]: Comment method

**Review by toryjleo — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Audio/Jukebox.cs (hunk 2 above)
Comment (LiamMillerSE) [reply Assets/Scripts/Audio/Jukebox.cs:105]: Removed

File: Assets/Scripts/Audio/Jukebox.cs (hunk 1 above)
Comment (LiamMillerSE) [reply Assets/Scripts/Audio/Jukebox.cs:19]: No longer used - removed

**Review by toryjleo — APPROVED** [review]

**Pull request merged**